src/components/TaskPreviewPanel.js

```diff
@@ -1,68 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
 import { Paper, Typography } from '@material-ui/core';
-import { makeStyles } from '@material-ui/styles';
-import TaskPreview from './TaskPreviewTable';
+import {
+  Contributions,
+  withHistory,
+  withModulesManager,
+} from '@openimis/fe-core';
+import { BENEFIT_PLAN_TASK_PREVIEW_TABLE_VALUE, TASKS_PREVIEW_CONTRIBUTION_KEY } from '../constants';
 
-const useStyles = makeStyles((theme) => ({
+const styles = (theme) => ({
   paper: theme.paper.paper,
   title: theme.paper.title,
-}));
+});
 
-const DUMMY_PREVIEW_ITEMS = [
+function TaskPreviewPanel({
+  intl,
+  classes,
+  rights,
+  formatMessage,
+  edited,
+}) {
+  const [taskTable, setTaskTable] = useState('');
```

Comment:
Use _EMPTY_STRING_ constant from _constants.js_ instead of raw empty string **' '**.

Reply:
done